primitive/uuid.go

```diff
@@ -26,11 +26,10 @@ const LengthOfUuid = 16
 
 type UUID [16]byte
 
-func (u *UUID) Clone() *UUID {
+func (u *UUID) DeepCopy() *UUID {
```

Comment:
This is the only DeepCopy method that had to be kept manual because the generator does not handle array types.